### ISSUE TITLE
Reading audio file takes more time

### ISSUE THREAD

**Issue by vinodatphp** [google/ringdroid#29]
In RingdroidEditActivity, loading audio file takes more time, but in your app audio loads within fraction of second, how

**Comment by yangpin** [comment]
Hi ,it is a great app . i have learned so much 
but i have the same question   
it takes to much time to read the file

**Comment by yangpin** [comment]
Do you find the reason？

**Comment by vinodatphp** [comment]
No, if I use older version of ringdroid library, it load faster.

**Comment by vinodatphp** [comment]
link to ringdroid library older version,
https://github.com/mitch000001/ringdroid?files=1

**Comment by yangpin** [comment]
Thanks

**Comment by yangpin** [comment]
i tried  the old version  ,i takes just a sec ,thank u so much

**Comment by kash8080** [comment]
but how can we improve the current version's reading speed? I don't want to go to an older release as it might have bugs.

**Comment by yangpin** [comment]
I think we can try to use the old CheapSoundFile in the new Version

**Comment by kash8080** [comment]
but doesn't that contain deprecated methods. ? you are using old version in your app .. is it working fine on all api versions?

**Comment by yangpin** [comment]
You can delete the deprecated methods,  the differences are SoundFile.SamplePlayer ZoomGesture.and so on.the waveFormView is working fine too.

**Comment by kash8080** [comment]
can you please post your working code.. that would really help..

**Comment by yangpin** [comment]
I use the old version currently .but i have tried it before...
u can just replace the SoudFile to CheapSoundFile... in the RingdroidEditActivity and WaveformView 
they all have the parameters,it is not hard to do this.

**Comment by kash8080** [comment]
ok tx.

**Comment by digitGrove** [comment]
after building the release apk the sound file was loading faster.. strange but don't know why there was a loading time difference between debug version and release version with no code changes.

**Comment by rahul107** [comment]
hello, i have same issue can anyone suggest me how to solve it?
I add CheapSoundFile instead of SoundFile but there are many errors raise and i am not able to resolve it.

Thanks
